src/components/NoteInput/NoteInput.scss

```diff
@@ -46,21 +46,22 @@ $note-input__input-right: 40px;
   }
 }
 
+.note-input__image-info,
 .note-input__add-button {
   all: unset;
-  cursor: pointer;
+  position: relative;
   display: flex;
-  flex-direction: column;
+  align-self: flex-start;
   padding: $spacing--xs 0;
+}
 
-  &:hover > .note-input__icon {
-    filter: brightness(1.2);
-    transform: scale(1.2);
-  }
+.note-input__add-button {
+  cursor: pointer;
 
-  &:focus > .note-input__icon {
-    border-radius: 100%;
-    outline: 2px solid rgba(var(--accent-color-rgb), 0.5);
+  &:hover > .note-input__icon--add {
+    filter: $brighten--slightly;
+    transform: scale(1.2);
```

Comment:
I think we should try to unify the hover styling across our elements. Therefore, I'd suggest that we discard the `transform: scale(1.2)`